content/news/belaz.en.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2018-09-10 10:00:00 +03:00
+title: VibroBox and "BELAZ-HOLDING" (OJSC "BELAZ") are planning to make mining trucks more effective
+description: VibroBox and "BELAZ-HOLDING" signed a cooperation agreement, after successful testing of the VibroBox hardware and software solution..
+keywords: VibroBox, Belaz, tests, cooperation, agreement, result, Minsk, Belarus, automatic, diagnostic, management, system, project, 2019
+share:
+  image: img/news/belaz/vibrobox-and-belaz.jpg
+---
+# VibroBox and "BELAZ-HOLDING" (OJSC "BELAZ") are planning to make mining trucks more effective
+
+{{<date>}}
+
+The mining industry operates a lot of critical and expensive equipment, downtime is extremely costly for the business. Manufacturers and consumers of mining equipment are interested in improving the reliability and extending the life of their equipment.
```

Comment:
There is a lot of critical and expensive equipment in the mining industry, which downtime is extremely costly for businesses.